packages/components/src/Filter/FilterBar/context/FilterBarContext.tsx

```diff
@@ -135,6 +146,21 @@ export const FilterBarProvider = <ValuesMap extends FiltersValues>({
     }
   }, [state])
 
+  useEffect(() => {
+    if (state.hasUpdatedValues) {
+      onValuesChange({ ...state.values })
+      dispatch({ type: "complete_update_values" })
+    }
+  }, [state])
```

Comment:
This appears to be a duplicate of the above pre-existing useEffect. Was it needed?

Reply:
Nup, must have been a bad rebase. I'll remove

Reply:
Done